src/styles.css

```
@@ -1,3 +1,7 @@
+body {
+  background: #e5efff;
```

généralement il faut mettre les couleurs css dans une variable.
```suggestion
 :host { 
   --default-background : #e5efff;
   }
```